### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by marcalmon**
## Summary

- Completed the navbar card.

## Screenshots

navbar

## Trello Card

https://trello.com/c/yYKfZ5eZ/13-navbar

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -8,37 +8,41 @@
...
+    <p class="nav-text">Developer Tools</p>
+  </div>
   <div class="content-title">
     <img src="./assets/triangle.svg" alt="" class="logo-title">
     <h1 class="title">Front-End Tools</h1>
   </div>
   <div class="content-tech">
     <div class="tech">
-      <img src="./assets/react_icon.svg" alt="" class="tech-image">
+      <img src="../maquetado/assets/react_icon.svg" alt="" class="tech-image">
```
Comment (pabloccid) [inline maquetado/index.html]: You dont need to reference `maquetado` folder.
You can just do `src="assets/react_icon.svg"`. Also, you should give this image an alt attribute.
Apply this comment to every image

File: maquetado/index.html [hunk 2]
```diff
@@ -8,37 +8,41 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
+  <div class="navbar">
+    <img class="nav-logo" src="./assets/wolox_logo.svg" alt="">
```
Comment (pabloccid) [inline maquetado/index.html]: Add an alt attribute. Every image should have one

File: scss/tech.scss [hunk 3]
```diff
@@ -35,6 +35,12 @@
       z-index: -1;
     }
 
+    &:hover::after {
```
Comment (pabloccid) [inline scss/tech.scss]: You dont need to make any change here, but notice that you included the changes of the PR #7 , when it wasnt needed

File: scss/application.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
 @import 'equalizer';
 @import 'tech';
 @import 'titles';
+@import 'navbar';
```
Comment (gabrielizalo) [inline scss/application.scss]: Remember sort attributes alphabetically.

File: maquetado/index.html [hunk 5]
```diff
@@ -8,70 +8,75 @@
...
+    <p class="nav-text">Developer Tools</p>
+  </div>
   <div class="content-title">
-    <img src="./assets/triangle.svg" alt="" class="logo-title">
+    <img src="./assets/triangle.svg" class="logo-title">
```
Comment (gabrielizalo) [inline maquetado/index.html]: This is a decorative image... It should have the `alt` attribute but empty.
Remember W3C rules:
_Guidelines for the alt text:_
- The text should describe the image if the image contains information
- The text should explain where the link goes if the image is inside an <a> element
- Use alt=“” if the image is only for decoration (edited)

File: maquetado/index.html (hunk 2 above)
Comment (gabrielizalo) [reply maquetado/index.html]: Remember to add the alternative text.

File: scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  padding: 0 20px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+
+
```
Comment (pabloccid) [inline scss/navbar.scss]: Remove this extra empty line

**Pull request merged**